http/src/main/resources/init-resources/startup.sh

```diff
@@ -228,13 +228,14 @@ else
           ${DOCKER_COMPOSE} -f ${JUPYTER_DOCKER_COMPOSE} stop
           ${DOCKER_COMPOSE} -f ${JUPYTER_DOCKER_COMPOSE} rm -f
           ${DOCKER_COMPOSE} -f ${JUPYTER_DOCKER_COMPOSE} up -d
-
-          log 'Copy Jupyter frontend notebook config...'
-          $GSUTIL_CMD cp ${JUPYTER_NOTEBOOK_FRONTEND_CONFIG_URI} /var
-          JUPYTER_NOTEBOOK_FRONTEND_CONFIG=`basename ${JUPYTER_NOTEBOOK_FRONTEND_CONFIG_URI}`
-          retry 3 docker exec -u root ${JUPYTER_SERVER_NAME} /bin/bash -c "mkdir -p $JUPYTER_HOME/nbconfig"
-          docker cp /var/${JUPYTER_NOTEBOOK_FRONTEND_CONFIG} ${JUPYTER_SERVER_NAME}:${JUPYTER_HOME}/nbconfig/
         fi
+
+        log 'Copy Jupyter frontend notebook config...'
```

Comment:
Why keep this here? It looks like it is exactly like the commands that were removed from the block above